src/main/java/edu/harvard/iq/dataverse/api/Dataverses.java

```diff
@@ -976,6 +976,8 @@ public Response listAssignments(@Context ContainerRequestContext crc, @PathParam
      */
 //    File tempDir;
 //
+//    TODO: Code duplicate in ThemeWidgetFragment. Maybe extract, make static and put some place else?
```

Comment:
Should you create and link to an issue if you create a TODO comment? I think there is a good chance that loose TODOs are never addressed after they have been added.

Reply:
Thanks for the comment. Yet, with the SPA on the horizon, I don't know if anyone will actually be spending any cycles on this. It will be ditched sooner than later. I'm going to resolve this conversation.